apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -58,6 +61,33 @@ dayjs.extend(customParseFormat);
 
 type Props = AvailabilityTeamPageProps | AvailabilityPageProps;
 
+export const locationKeyToString = (location: LocationObject, t: TFunction) => {
```

Comment:
Determine what to display to the user

Reply:
@zomars do we want to extract this eventually into app store? otherwise each video integration needs to be added here

Reply:
i think this can be solved without a switch

Reply:
opened a ticket for a future PR: https://github.com/calcom/cal.com/issues/2753

Reply:
Can we use https://github.com/calcom/cal.com/blob/main/apps/web/lib/locationOptions.tsx here?